### ISSUE TITLE
covidcast::fips_to_name for Puerto Rico

### ISSUE THREAD

**Issue by jacobbien** [cmu-delphi/covidcast#446]
The FIPS 72000 is recognized in `fips_to_abbr` but not in `fips_to_name`:

```
> covidcast::fips_to_abbr("72000")
72000 
 "PR" 
> covidcast::abbr_to_fips("PR")
     PR 
"72000" 
```

But then...
```
> covidcast::fips_to_name("72000")
72000 
   NA 
```

**Comment by statsmaths** [comment]
This seems fixed with #496. Is there are reason it is still open?

**Comment by capnrefsmmat** [comment]
Probably not
